app/controllers/api/v1/users_controller.rb

```diff
@@ -3,9 +3,10 @@ class Api::V1::UsersController < Api::ApplicationController
 
   # POST /api/v1/users
   def create
-    @user = User.new(secure_params.merge(
+    @user = User.where(email: email).first_or_initialize
+    @user.update(secure_params.merge(
       password: User.generate_password,
-      email: email
+      personal_emails: @user.personal_emails.concat(secure_params[:personal_emails])
```

Comment:
Should it be possible to remove personal emails?

Reply:
That has not been requested.